### DISCUSSION TITLE
The application does not start

### DISCUSSION THREAD

**Discussion by vitaliy4us** [cypress-io/cypress-realworld-app#1073] [General]
The application does not start. Steps to reproduce:
1. Clone the project from GitHub on your machine.
2. Run **yarn install** as required.
3. Run **yarn dev**  as required.
**Expected result:** the application is started an is accessible on localhost:3000
**Actual result:** The browser starts working but its window is blank on localhost:3000. If being reloaded the browser window displays some errors:

![errors](https://user-images.githubusercontent.com/12917391/130180812-2875e056-b288-4109-a48f-c306cac81d80.png)

**Error:** Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: null.
Check the render method of `TransitionGroup`.

**Module anonymous**
` 92 |     document.getElementById("root")
  93 |   );
  94 | } else {
> 95 |   ReactDOM.render(
  96 |     <Router history={history}>
  97 |       <ThemeProvider theme={theme}>
  98 |         <App />`

**Module ../src/index.tsx**
__webpack_require__
`853 | 
  854 | __webpack_require__.$Refresh$.init();
  855 | try {
> 856 | 	modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId));
      | ^  857 | } finally {
  858 | 	__webpack_require__.$Refresh$.cleanup(moduleId);
  859 | }`

**fn**
` 147 | 		);
  148 | 		hotCurrentParents = [];
  149 | 	}
> 150 | 	return __webpack_require__(request);
      | ^  151 | };
  152 | var ObjectFactory = function ObjectFactory(name) {
  153 | 	return {`

**1**
__webpack_require__
`  853 | 
  854 | __webpack_require__.$Refresh$.init();
  855 | try {
> 856 | 	modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId));
      | ^  857 | } finally {
  858 | 	__webpack_require__.$Refresh$.cleanup(moduleId);
  859 | }`

**checkDeferredModules**
`42 | 	}
  43 | 	if(fulfilled) {
  44 | 		deferredModules.splice(i--, 1);
> 45 | 		result = __webpack_require__(__webpack_require__.s = deferredModule[0]);
     | ^  46 | 	}
  47 | }
  48 | `

**Array.webpackJsonpCallback [as push]**
`29 | 	deferredModules.push.apply(deferredModules, executeModules || []);
  30 | 
  31 | 	// run deferred modules when all chunks ready
> 32 | 	return checkDeferredModules();
     | ^  33 | };
  34 | function checkDeferredModules() {
  35 | 	var result;`

**Comment by vitaliy4us** [comment]
Is anybody there to provide at least a comment regarding the issue blocking the application?

**Comment by vitaliy4us** [comment]
Please comment  the issue in any way.

**Comment by robertguss** [comment]
Hi @vitaliy4us which branch are you on when you try to do `yarn install` and `yarn dev` you need to be on the `develop` branch. Not `master`.

Please give that a try and let me know if that helps to get things up and running for you.

I just pulled the latest version of the `develop` branch and can confirm that everything is working. 

Also, which version of Yarn are you using? Make sure you are using a version of Yarn 1 and not 2.

**Comment by vitaliy4us** [comment]
Thank you @robertguss for updating now the application starts properly.
